README.md

```diff
@@ -363,7 +371,7 @@ method takes one argument:
 #### `map` method
 
 ```hs
-map :: Functor f => f a ~> (a -> b) -> f b
+map :: forall a b . Functor f => f a ~> (a -> b) -> f b
```

Comment:
PureScript appears not to include a space before the `.`. I suggest we follow suit.

Reply:
Fixed.

Reply:
(Space is allowed in PS before the dot btw)

Reply:
Yes, allowed in Haskell as well, at least in some example code, but it's much more rare, it seems.